src/Cmi5Statements.ts

```diff
@@ -36,6 +36,21 @@ export function Cmi5DefinedStatement(
   ctx: LaunchContext,
   statement: Partial<Statement>
 ): Statement {
+  // 10.2.2 Launch Mode - Restrict cmi5 defined statements in Browse/Review
+  const verbId = statement.verb?.id || statement.verb;
+  const isTerminated = verbId === Cmi5DefinedVerbs.TERMINATED?.id || verbId === Cmi5DefinedVerbs.TERMINATED;
+  const isInitialized = verbId === Cmi5DefinedVerbs.INITIALIZED?.id || verbId === Cmi5DefinedVerbs.INITIALIZED;
```

Comment:
@LGipsonPersonal Can I ask why we're trying both the verb ID or falling back the entire verb? I think just checking the ID is what we will need here, as the display property of the verb isn't used as an identifier and can depend on the language set.

What instances can you see where there wouldn't be a verb ID?